lib/pipeline/core/stage_action_runner.cpp

```diff
@@ -32,14 +32,18 @@ SharedTranspilerSessionResult runStageAction(StringRef stageName, SharedTranspil
     SPDLOG_INFO("start: {}", stageName);
     SPDLOG_TRACE("input source:\n{}\n", source);
 
-    Twine toolName = "clang";//stageName;
+    Twine toolName = "clang";  // stageName;
 
     auto cppFileNamePath = input.sourcePath;
     auto cppFileName = std::string(cppFileNamePath.replace_extension(".cpp"));
 
     // TODO get this info from user input aka json prop file
-    std::vector<std::string> args = {
-        "-std=c++17", "-Wno-extra-tokens", "-Wno-invalid-pp-token", "-fparse-all-comments", "-I.", getISystemOpt()};
+    std::vector<std::string> args = {"-std=c++17",
+                                     "-Wno-extra-tokens",
+                                     "-Wno-invalid-pp-token",
+                                     "-fparse-all-comments",
```

Comment:
Do we have any hooks for Comment parsing and re-insertion? If not, this option may be unused.